### PR TITLE
Allow invalid email

### PR DESCRIPTION
**Pull request by johnwatson484**

### CODE REVIEW

File: app/monitoring/get-outstanding-deliveries.js [hunk 1]
```diff
@@ -1,7 +1,14 @@
 const db = require('../data')
 
 const getOutstandingDeliveries = async () => {
-  return db.delivery.findAll({ where: { completed: null } })
+  return db.delivery.findAll({
+    where: {
+      completed: null,
+      reference: {
+        [db.Sequelize.Op.ne]: null
```
Comment (johnwatson484) [inline app/monitoring/get-outstanding-deliveries.js:8]: Don't monitor deliveries not sent to Notify

File: app/publishing/index.js [hunk 2]
```diff
@@ -2,11 +2,16 @@ const { EMAIL } = require('../methods')
...
 const publish = require('./publish')
 const saveRequest = require('./save-request')
+const validateEmail = require('./validate-email')
 
 const publishStatement = async (request) => {
-  const personalisation = getPersonalisation(request.scheme.name, request.scheme.shortName, request.scheme.year, request.scheme.frequency, request.businessName)
-  const response = await publish(request.email, request.filename, personalisation)
-  await saveRequest(request, response.data.id, EMAIL)
+  let response
+  if (validateEmail(request.email)) {
```
Comment (johnwatson484) [inline app/publishing/index.js:9]: Only send to Notify if email

File: app/publishing/index.js [hunk 3]
```diff
@@ -2,11 +2,16 @@ const { EMAIL } = require('../methods')
...
+    const personalisation = getPersonalisation(request.scheme.name, request.scheme.shortName, request.scheme.year, request.scheme.frequency, request.businessName)
+    response = await publish(request.email, request.filename, personalisation)
+    console.log(`Statement published: ${request.filename}`)
+  }
+  await saveRequest(request, response?.data.id, EMAIL)
```
Comment (johnwatson484) [inline app/publishing/index.js:14]: Save detail either way

**Review by AbidemiAdio — APPROVED** [review]
QED

**Pull request merged**